model-optimizer/mo/ops/pooling.py

```diff
@@ -23,6 +23,25 @@
 from mo.utils.error import Error
 
 
+class PoolingV2(Op):
+    op = 'PoolingV2'
+    enabled = True
```

Comment:
Must be `False`. We don't want to use `Ops` as extractors anymore.

Reply:
fixed